src/bundler.ts

```diff
@@ -30,32 +30,35 @@ import {FSUrlLoader} from 'polymer-analyzer/lib/url-loader/fs-url-loader';
 import constants from './constants';
 import * as matchers from './matchers';
 import PathResolver from './pathresolver';
-import ASTUtils from './ast-utils';
-
-
-function buildLoader(config: any) {
-  const abspath: string = config.abspath;
-  const excludes = config.excludes;
-  const fsResolver = config.fsResolver;
-  const redirects = config.redirects;
-  let root = abspath && path.resolve(abspath) || process.cwd();
-  let loader = new FSUrlLoader(root);
-  // TODO(garlicnation): Add noopResolver for external urls.
-  // TODO(garlicnation): Add redirectResolver for fakeprotocol:// urls
-  // TODO(garlicnation): Add noopResolver for excluded urls.
-  return loader;
+import * as ASTUtils from './ast-utils';
+
+
+export interface Options {
+  abspath?: string;
+  addedImports?: string[];
+  analyzer?: Analyzer;
+  excludes?: string[];
+  implicitStrip?: boolean;
+  inlineCss?: boolean;
+  inlineScripts?: boolean;
+  inputUrl?: string;
+  redirects?: string[];
+  stripComments?: boolean;
+  stripExcludes?: string[];
 }
 
+
 class Bundler {
-  constructor(opts: any) {
+  constructor(opts: Options) {
+    this.analyzer = opts.analyzer!;
     // implicitStrip should be true by default
     this.implicitStrip =
         opts.implicitStrip === undefined ? true : Boolean(opts.implicitStrip);
     this.abspath = (String(opts.abspath) === opts.abspath &&
                     String(opts.abspath).trim() !== '') ?
         path.resolve(opts.abspath) :
-        null;
-    this.pathResolver = new PathResolver(this.abspath);
+        undefined;
```

Comment:
There is a big ugly PathResolver module cargo'd in from old vulcanize that probably needs to be refactored to take analyzer's resolver into account, but it apparently uses abspath.  Will replace or update it once I have all the tests covering path resolution working.